### PR TITLE
Use URL of repository

### PR DESCRIPTION
**Pull request by c-martinez**
This PR modifies the action to take `REPOSITORY` as an input and passes it to `howfairis` script.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,4 +6,4 @@ WORKDIR /data
 
 RUN pip install -e .
 
-CMD ["howfairis"]
+CMD howfairis "https://github.com/${INPUT_REPOSITORY}"
```
Comment (edeediong) [inline Dockerfile:9]: Nice work Carlos! For best practices in writing Dockerfiles, cmd is better represented by

```
CMD [ "howfairis", "https://github.com/${INPUT_REPOSITORY}"]
```

**Comment by jspaaks** [comment]
Hi @c-martinez, is it OK with you if I update this branch a bit so it is in line with the recent updates we did on ``master``?

**Comment by c-martinez** [comment]
Hey @jspaaks -- yes, please go ahead.

File: Dockerfile [hunk 2]
```diff
@@ -1,9 +1,11 @@
 FROM python:3.8-alpine
 
+RUN apk add git
+
 RUN mkdir /data
 COPY . /data/
```
Comment (c-martinez) [inline Dockerfile:6]: If installing directly from github, do you still need to add files from `.` ?

**Comment by jspaaks** [comment]
Could you take another look at this @c-martinez ? Thanks!

**Review by jspaaks — APPROVED** [review]
Nice work Carlos

**Comment by c-martinez** [comment]
Looks good to me -- thanks @jspaaks for all the updates. Can we merge it now?

**Comment by jspaaks** [comment]
> Looks good to me -- thanks @jspaaks for all the updates. Can we merge it now?

Yes please go ahead a press the green button

**Pull request merged**